test/clj/rems/testing_util.clj

```diff
@@ -54,3 +54,8 @@
                                         (organizations/get-all-organization-roles ~user)
                                         (applications/get-all-application-roles ~user))]
      ~@body))
+
+(defmacro with-fake-login-users [users & body]
+  `(with-redefs [rems.auth.fake-login/get-fake-login-users (constantly ~users)]
+     ~@body))
+
```

Comment:
i need to pay a little bit attention to the empty new lines, as for some reason my VSCode stopped correctly adding new lines at the end of the file. could be a broken plugin.

![Screenshot 2022-01-25 at 17 55 15](https://user-images.githubusercontent.com/794087/151011661-49a31b99-6102-48db-a1e9-2439d92c3d0d.png)